pkgs/development/python-modules/pyathena/default.nix

```diff
@@ -0,0 +1,41 @@
+{ pkgs
+, buildPythonPackage
+, fetchPypi
+, sqlalchemy
+, boto3
+, botocore
+, pandas
+, tenacity
+}:
+
+let
+  pname = "PyAthena";
+  version = "2.3.0";
+in buildPythonPackage {
+  inherit pname version;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "08fl653yayvqi991zvcai5ifcxwy9ip6xh0cr3lbimggjnjgwsl5";
+  };
+
+  checkInputs = [
+    sqlalchemy
```

Comment:
Are we missing pytestCheckHook here?

Reply:
Maybe that is the reason why no tests are being run

Reply:
Tests depend on a working AWS athena instance, it seems: https://github.com/laughingman7743/PyAthena/#testing

Short of mocking it, I'd just leave out tests, and replace with a comment.

Reply:
We worked through the tests extensively and tried to enable them. It seems that 1. it is a lot of brittle work 2. most of them won't work because of missing AWS access during `checkPhase`, so we're better off disabling them completely.